lua/block/block.lua

```diff
@@ -73,19 +73,33 @@ end
 -- a func called tab_to_space that converts each tab to tabstop amount of spaces
 ---@param mts_node MTSNode
 local function color_mts_node(mts_node, lines)
+    local offset = vim.fn.winsaveview().leftcol
     for row = mts_node.start_row, math.min(#lines - 1, mts_node.end_row) do
         -- Set the padding at the end of the line
         local str_len = string.len(lines[row + 1])
-        vim.api.nvim_buf_set_extmark(0, ns_id, row, 0, {
-            virt_text = { { string.rep(" ", mts_node.end_col - str_len + mts_node.pad),
-                "bloc" .. mts_node.color % nest_amount } },
-            virt_text_win_col = str_len,
-            priority = 100 + mts_node.color,
-        })
+        if offset+1 <= str_len then
+          pcall(vim.api.nvim_buf_set_extmark,
```

Comment:
I would assume it'd be possible to do this without pcall. I'll look in to it!

Reply:
Yes, avoid using `pcall` should be preferred. Do I need to remove it from this PR and create a new issue of the error?

Reply:
What was the error?

Reply:
I forgot, I didn't record it. It said something is out of range IIRC.

Reply:
Found it :) I just created a standalone issue for it, see #17.